demo/src/app/components/+popover/demos/toggle-manual/toggle-manual.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-popover-toggle-manual',
+  templateUrl: './toggle-manual.html'
+})
+export class DemoPopoverToggleManualComponent {
```

Comment:
change the name to `DemoPopoverTriggerByInput` (and selector, filenames and folder name as well)